mmv1/templates/terraform/examples/backend_service_ip_address_selection_policy.tf.tmpl

```diff
@@ -0,0 +1,5 @@
+resource "google_compute_backend_service" "{{$.PrimaryResourceId}}" {
```

Comment:
I don't seen these examples being picked up anywhere (to be used in the docs and/or tests). Did you perhaps forget to add `examples` blocks in the yaml files?

Reply:
Ah, indeed, they should be added now.